.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Hi! Thank you for helping out with Embedded Graphics development! Please:
+
+* [ ] Add a `CHANGELOG.md` entry in the **Unreleased** section under the appropriate heading
+* [ ] Run `rustfmt` on the project - please check CI for failures
```

Comment:
> please check CI for failures

I think it could be clearer that this is only possible after the PR is already opened. All other checks are supposed to be done before opening the PR.